backend/models/userModel.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcrypt");
+
+
+const userSchema = mongoose.Schema(
+  {
+    name: {
```

Comment:
The name shouldn't be required in the schema no? (Since for the login and registration, we're only taking in the email and password)

Reply:
We can technically still leave this in here (if we were to extend functionality) although the frontend will only be supplying the email and password. (So this field wouldn't be required)

Reply:
Fixed: Changes are reflected in recent commits